ckanext/collaborators/views/package.py

```diff
@@ -0,0 +1,91 @@
+from flask.views import MethodView
```

Comment:
See comment above, this file should move to `ckanext/collaborators/blueprint.py`.

Reply:
This import is in `package.py` because I need to inherit the class to create the views.